rhel8/profiles/ospp.profile

```diff
@@ -9,846 +9,973 @@ description: |-
-    ### objectives for the operational environment as described in the ST
-    - installed_OS_is_vendor_supported
-    - installed_OS_is_FIPS_certified
-    - grub2_audit_argument
-    - grub2_audit_backlog_limit_argument
-    - service_auditd_enabled
-    - rpm_verify_hashes
-    - selinux_all_devicefiles_labeled
-    - selinux_confinement_of_daemons
-    - selinux_policytype
-    - selinux_state
-    - audit_rules_immutable
-    - var_selinux_policy_name=targeted
-    - var_selinux_state=enforcing
-    - ensure_redhat_gpgkey_installed
-    - ensure_gpgcheck_globally_activated
-    - ensure_gpgcheck_never_disabled
-    - ensure_gpgcheck_local_packages
+    ## RHEL 8 CCE-81020-0: Enable Kernel Paremeter to Log Martian Packets on all IPv4 Interfaces by Default
+    - sysctl_net_ipv4_conf_default_log_martians
+
+    ## RHEL 8 CCE-81021-8: Enable Kernel Parameter to Use Reverse Path Filtering on all IPv4 Interfaces
+    - sysctl_net_ipv4_conf_all_rp_filter
+
+    ## RHEL 8 CCE-81022-6: Enable Kernel Parameter to Use Reverse Path Filtering on all IPv4 Interfaces by Default
+    - sysctl_net_ipv4_conf_default_rp_filter
+
+    ## RHEL 8 CCE-81023-4: Enable Kernel Parameter to Ignore Bogus ICMP Error Responses on IPv4 Interfaces
+    - sysctl_net_ipv4_icmp_ignore_bogus_error_responses
+
+    ## RHEL 8 CCE-80922-8: Enable Kernel Parameter to Ignore ICMP Broadcast Echo Requests on IPv4 Interfaces
+    - sysctl_net_ipv4_icmp_echo_ignore_broadcasts
+
+    ## TO DO: NEED SCAP RULE
+    #echo "net.ipv6.icmp.echo_ignore_all = 0" >> $CONFIG
+
+    ## RHEL 8 CCE-81024-2: Disable Kernel Parameter for IP Forwarding on IPv4 Interfaces
+    - sysctl_net_ipv4_ip_forward
 
-    #######################################################
-    # Appendix A Optional Requirements
+    ## RHEL 8 CCE-80923-6: Enable Kernel Parameter to Use TCP Syncookies on IPv4 Interfaces
+    - sysctl_net_ipv4_tcp_syncookies
 
-    #######################################################
-    ## FCS_TLSC_EXT.4 TLS Client Protocol
+    #################################################################
+    ## File System Settings
+    #################################################################
 
-    ### FCS_TLSC_EXT.4.1: https://www.niap-ccevs.org/MMO/PP/-424-/#FCS_TLSC_EXT.4.1
-    ### The OS shall support mutual authentication using X.509v3 certificates
+    ## RHEL 8 CCE-81027-5: Enable Kernel Parameter to Enforce DAC on Hardlinks
+    - sysctl_fs_protected_hardlinks
 
-    #######################################################
-    ## FDP_IFC_EXT.1 Information flow control
+    ## RHEL 8 CCE-81030-9: Enable Kernel Parameter to Enforce DAC on Symlinks
+    - sysctl_fs_protected_symlinks
 
-    ### FDP_IFC_EXT.1.1: https://www.niap-ccevs.org/MMO/PP/-424-/#FDP_IFC_EXT.1.1
-    ### The OS shall [selection:
-    ###     - provide an interface which allows a VPN client to protect
-    ###       all IP traffic using IPsec,
-    ###     - provide a
```

Comment:
This should be handled by system crypto policies instead. But none of the currently available crypto policies in RHEL8 complies with CC requirements. @yuumasato Do you think the rules can be enabled here until it is handled on crypto policies side, or should we wait?

Reply:
I don't know why GitHub UI show this IPSec, because the comment isn't related to IPSec, but is related to the selection of SSHD rules sshd_use_approved_macs and sshd_use_approved_ciphers.